packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -0,0 +1,59 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  box-sizing: border-box;
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: 1rem;
+  font-style: normal;
+  font-weight: 400;
+  letter-spacing: 0;
+  line-height: $base-line-height;
+  font-feature-settings: "lnum", "pnum";
+  text-decoration-skip-ink: all;
+
+  // Catch-all border-box
+  /* stylelint-disable selector-max-universal */
+  * {
+    box-sizing: border-box;
+
+    &::before,
+    &::after {
+      box-sizing: inherit;
+    }
+  }
+  /* stylelint-enable selector-max-universal */
+
+  /* stylelint-disable selector-max-type */
+  // HTML5 display-role reset for older browsers
+  article:not([class]),
+  aside:not([class]),
+  details:not([class]),
+  figcaption:not([class]),
+  figure:not([class]),
+  footer:not([class]),
+  header:not([class]),
+  hgroup:not([class]),
+  menu:not([class]),
+  nav:not([class]),
+  section:not([class]) {
```

Comment:
I'm not sure I understand how this `:not([class])` is intended to be used? Can you explain further?

Can you explain further why we need to target elements from Box like this? Do we have regressions if this code isn't included? Where?

As an example, if we need a `menu` that has `display: block` for all browsers we should build a `<Menu />` in the future to solve for this where the styles live alongside, rather than letting these styles live here in Box. The same concept is true of `svg` and `img` styles below. I'm not sure I follow why we need to target specific DOM elements here

Reply:
The intent was to get expected behavior for HTML 5 elements in older browsers but only if they were unclassed. This rule was originally trumping an applied class and causing our VRT error.

Pulling all this as IE should be fine without these rules.